ros/ros1/OdometryServer.cpp

```diff
@@ -46,6 +46,7 @@ namespace kiss_icp_ros {
 
 OdometryServer::OdometryServer(const ros::NodeHandle &nh, const ros::NodeHandle &pnh)
     : nh_(nh), pnh_(pnh) {
+    bool alias_transform;
```

Comment:
Do we even need this duplicate variable? Why not using `publish_odom_tf_` instead?

Reply:
these are 2 different options - 
turning off alias transforms disables publishing the base_link to sensor frames
which is independent from publishing the odom message

e.g.
some users might want odom published but already have a base_link -> sensor frame defined in URDF
other users might not have tf defined (and need this published, arguable this is a strange behavior in general for ROS in my opinion) but _do_ want the odom frame published